views/js/review/plugins/navigation/review-panel/tpl/fizzyPanel.tpl

```diff
@@ -0,0 +1,22 @@
+<div class="review-panel fizzy">
+    <header class="review-panel-header">
+        <div class="review-panel-title">{{headerTitle}}</div>
+        {{#if header}}
+            <div class="review-panel-score-header">
+            {{#with header}}
+                <span class="review-panel-label">{{label}}</span>
+                <span class="review-panel-score">{{score}}</span>
+            {{/with}}
```

Comment:
`#with` syntax is specific to Handlebars 4. We'd better refactor this because we don't support it yet.

Reply:
My mistake. Special syntax `{{#with header as |foo|}}` was added in Handlebars 4. We don't use it.